lib/internal/http2/compat.js

```diff
@@ -158,13 +160,12 @@ class Http2ServerRequest extends Readable {
     stream.on('end', onStreamEnd);
     stream.on('error', onStreamError);
     stream.on('close', onStreamClosedRequest);
-    stream.on('aborted', onStreamAbortedRequest.bind(this));
```

Comment:
Any `bind` in here is expensive. I think only binding the most crucial things that are almost guaranteed to run (that being `onfinish`) is smart.